README.md

```diff
@@ -8,6 +8,20 @@
 
 _TAXonomic Profile Aggregation and STAndardisation_
 
+## About
+
+The main purpose of taxpasta is to _standardise_ taxonomic profiles created by a
+range of bioinformatics tools. We call those tools taxonomic profilers. They
+each come with their own particular tabular output format. Relative abundances
+in read counts, fractions, or percentages. Any number of columns with additional
+information. We therefore decided to take [the lessons
+learnt](https://xkcd.com/927/) to heart and provide our own solution to deal
+with this pasticcio. With taxpasta you can ingest all of those formats and, at
```

Comment:
What is pasticcio?

Reply:
Italian informally means 'mess'. I thought it fits well with taxpasta.

Reply:
haha love it!

Reply:
Pasticcio or greek lasagna 😆